SRT to VTT Converter/MainWindow.xaml.cs

```diff
@@ -211,8 +211,8 @@ private void BackgroundWorker_RunWorkerCompleted(object sender, RunWorkerComplet
 
 		private void Convert(string sFilePath)
 		{
-			using (var strReader = new StreamReader(sFilePath))
-			using (var strWriter = new StreamWriter(sFilePath.Replace(".srt", ".vtt")))
+			using (var strReader = new StreamReader(sFilePath, Encoding.GetEncoding(1256)))
```

Comment:
Even though this fixes the conversion with files that have a Windows-1256 encoding, it also causes Unicode/UTF-8 files to be converted incorrectly. The best solution would be to detect the file's encoding beforehand and then read the file using that encoding, but I haven't been able to find any simple solutions to do that (although if you found one that would be fantastic).

I highly recommend converting your files to UTF-8 before converting them with the srt-to-vtt-converter or using [srt-to-vtt-cl](https://github.com/nwoltman/srt-to-vtt-cl) instead.